rust/feature-flags/src/api/request_handler.rs

```diff
@@ -174,7 +178,7 @@ pub async fn process_request(context: RequestContext) -> Result<FlagsResponse, F
     // NB don't charge if all the flags are survey targeting flags
     {
         if let Err(e) = increment_request_count(
-            context.state.redis.clone(),
+            context.state.redis_writer.clone(),
```

Comment:
this tracks `/decide` (now `/flags` but still called `decide`) requests, so it's a write.